orchestra/migrations/0019_auto_20151007_1846.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
This migration drops old fields and makes the friendly nullable fields non-nullable once the data is all in place.